piker/brokers/kraken.py

```diff
@@ -124,11 +124,11 @@ class Trade(BaseModel):
     Trade class that helps parse and validate ownTrades stream
 
     '''
-    reqid: str # kraken order transaction id
-    action: str # buy or sell
-    price: str # price of asset
-    size: str # vol of asset
-    broker_time: str # e.g GTC, GTD
+    reqid: str  # kraken order transaction id
```

Comment:
@iamzoltan fyi this is all the whitespace stuff you're missing bc no linter 😉

Reply:
no doubt. I need to get that up too haha